src/__tests__/posthog-core.js

```diff
@@ -68,7 +69,8 @@ describe('posthog core', () => {
             compression: {},
             __captureHooks: [],
             rateLimiter: {
-                isRateLimited: () => false,
+                isServerRateLimited: () => false,
+                isCaptureRateLimited: () => false,
```

Comment:
nit: since `capture` is our server endpoint the naming felt confusing to me. Could we rename to `isClientRateLimited` or `isSnippetRateLimited`?

Reply:
Would have to be `isClientCaptureRateLimited` as this only applies to capture calls...